api/client/network.go

```diff
@@ -222,7 +224,7 @@ func consolidateIpam(subnets, ranges, gateways []string, auxaddrs map[string]str
 				return nil, fmt.Errorf("multiple overlapping subnet configuration is not supported")
 			}
 		}
-		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}}
+		iData[s] = &network.IPAMConfig{Subnet: s, AuxAddress: map[string]string{}, Options: options}
```

Comment:
Agree. If we've not missed any side effects and others also agree, I would leave it as is: User inputted per-network ipam options will be inherited by each network's subnet.